openfisca_aotearoa/variables/entitlements/income_tax/best_start.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+
+# Import from openfisca-core the common python objects used to code the legislation in OpenFisca
+from openfisca_core.model_api import *
+# Import the entities specifically defined for this tax and benefit system
+from openfisca_aotearoa.entities import Person, Family
+from numpy import datetime64
+
+
+class income_tax__caregiver_eligible_for_best_start_tax_credit(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = MONTH
+    label = u'Boolean for if a Person is classified as eligible for best start tax credit'
+    reference = "http://legislation.govt.nz/bill/government/2017/0004/15.0/DLM7512349.html"
+
+    def formula(persons, period, parameters):
+        qualifies = persons("income_tax__qualifies_for_entitlements_under_family_scheme", period)
+        family_is_eligible = persons.family("income_tax__family_has_children_eligible_for_best_start", period)
+        return qualifies * family_is_eligible
+
+
+class income_tax__family_has_children_eligible_for_best_start(Variable):
+    value_type = bool
+    entity = Family
+    definition_period = MONTH
+    label = u'Boolean for if a family is classified as eligible for best start tax credit'
```

Comment:
These “Boolean for” are not understandable by non-programmers, yet don't provide value for programmers since `value_type = bool` is already there (and displayed in the Legislation Explorer).
I would recommend to describe variables in terms of what they model in legislation, not their implementation: “Is this family eligible for Best Start tax credit” or “Indicates whether this family is eligible for Best Start tax credit” are two good candidates to me 🙂

> This will become more obvious over time when using the Legislation Explorer to read through the model.